### PR TITLE
feat: Implement oracles filtering

### LINKED ISSUE #160: Implement oracles filtering
### Please describe the problem that your request should resolve.
Implement oracles table filtering once https://github.com/aeternity/ae_mdw/issues/1272 is released

and reflect the selected value in the url, as we do with transactions.

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #160 

## Demo

https://github.com/aeternity/aescan/assets/15363559/e0f4965e-ffdb-4191-b2ac-c41740fac43b

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] add oracle filtering

### CODE REVIEW

File: src/components/TransactionsPanel.vue [hunk 1]
```diff
@@ -55,12 +55,12 @@ async function loadNextTransactions() {
   await fetchTransactions(transactions.value.next)
 }
 
-const loadTransactions = () => {
+async function loadTransactions() {
```
Comment (janmichek) [inline src/components/TransactionsPanel.vue:58]: Unrelated to issue. consistency fixes in this file

**Review by Liubov-crypto — COMMENTED** [review]
Filtering is working good. 

But I have a question: It seems to me that order is broken here, correct me if I'm wrong: 
https://github.com/aeternity/aescan/assets/69896204/9de12a19-3dab-4ecc-99be-0ecc80647606

or it is sorted by Expiration date and not by Registered? or sorting by the date isn't implemented yet?

**Comment by Liubov-crypto** [comment]
LGTM

**Review by lukeromanowicz — APPROVED** [review]
Looks and works really nice. Well done @janmichek :rocket:

**Review by michele-franchi — CHANGES_REQUESTED** [review]
There seems to be an issue with history / navigation.
If you navigate to the oracles page and select "Inactive" in the filter and then go back you are redirected to "/oracles?state=active" and if you go back again you are redirected to "/oracles".

Could you check it?

**Comment by janmichek** [comment]
> There seems to be an issue with history / navigation. If you navigate to the oracles page and select "Inactive" in the filter and then go back you are redirected to "/oracles?state=active" and if you go back again you are redirected to "/oracles".
> 
> Could you check it?

Fixed

**Review by Liubov-crypto — APPROVED** [review]
Fixed, LGTM

**Review by michele-franchi — APPROVED** [review]
Good job

**Pull request merged**